cli/command/network/connect.go

```diff
@@ -56,7 +63,23 @@ func runConnect(dockerCli *command.DockerCli, options connectOptions) error {
 		},
 		Links:   options.links.GetAll(),
 		Aliases: options.aliases,
+		DriverOpts: driverOpts,
 	}
 
 	return client.NetworkConnect(context.Background(), options.network, options.container, epConfig)
 }
+
+func convertDriverOpt(opts []string) (map[string]string, error) {
+	driverOpt := make(map[string]string)
+	for _, opt := range opts {
+		parts := strings.SplitN(opt, "=", 2)
+		if len(parts) != 2 {
+			err := fmt.Errorf("invalid key value pair format in driver options")
+			return nil, err
+		}
+		key := strings.TrimSpace(strings.ToLower(parts[0]))
+		value := strings.TrimSpace(strings.ToLower(parts[1]))
```

Comment:
Are all options and values lowercase? If not needed, I'd rather not transform options that were passed by the user (in case some particular driver has options that are case sensitive)

Better to have the daemon return an error if an option is not valid

(I realize I didn't check the implementation for services, we may want to consider doing the same there)